python/telemetry/snet.py

```diff
@@ -168,10 +100,95 @@ def _decode(self, obj, context, path=None):
     'TargetData_ControlIsActive' / Int8ul  # flag, really
     )
 
+LTUFrameHeader = BitStruct(
+    'SrcId' / BitsInteger(7),
+    'DstId' / BitsInteger(7),
+    'FrCntTx' / BitsInteger(4),
+    'FrCntRx' / BitsInteger(4),
+    'SNR' / BitsInteger(4),
+    'AiTypeSrc' / BitsInteger(4),
+    'AiTypeDst' / BitsInteger(4),
+    'DfcId' / BitsInteger(2),
+    'Caller' / Flag,
+    'Arq' / Flag,
+    'PduTypeId' / Flag,
+    'BchRq' / Flag,
+    'Hailing' / Flag,
+    'UdFl1' / Flag,
+    'PduLength' / BitsInteger(10),
+    'CRC13' / BitsInteger(13),
+    'CRC5' / BitsInteger(5),
+    Padding(2)
+    )
+
+# For reference:
+#
+# To send mailbox messages
+# FCIDMajor = 56  (COM0)
+# FCIDSub = 34 || 134  (COM1)
```

Comment:
This does not seem to match the `Switch()` below?

Reply:
my idea with this comment was to document, in case someone was curious, what were the `FCIDMajor` and `FCIDSub` values if someone happens to implement an `framer` (?). The values in the `Switch()` are the relevant IDs for the deframe action. Do you think this needs to be removed or put somewhere else instead?

Reply:
Oh, so you mean that these are the FCIDMajor and FCIDMinor used in the uplink? It isn't clear from the comment (nor for your response, either; a framer could make frames identical to those transmitted by the satellite, or make uplink frames).

Reply:
yes, these `FCID*` are for uplink and yes, makes sense the uplink can be added to the framer too, I'm using gr-satellites to deframe payloads from the satellite to ground only, so for that reason, I was only scoping the changes to downlink.

I will add it.